.gitignore

```diff
@@ -0,0 +1,20 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
This file is better suited for a global .gitignore file on your machine, or the `exclude` file in `.git/info/exclude`. The `.gitignore` file for the project should only contain artifacts relevant to the project.